src/screens/TopicPageScreen/index.js

```diff
@@ -441,34 +443,8 @@ const TopicPageScreen = (props) => {
           showSearchBar={true}
           snap
           contentInsetAdjustmentBehavior={feeds?.length > 1 ? 'automatic' : 'never'}
-          onMomentumScrollEnd={(event) => {
-            onWillSendViewPostTime(event, feeds);
-            fetchNextFeeds(event);
-          }}
+          onMomentumScrollEnd={(event) => onWillSendViewPostTime(event, feeds)}
         />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The removal of the `fetchNextFeeds(event)` function call in the `onMomentumScrollEnd` event handler might affect the pagination logic if it was responsible for loading more feeds when the user reaches the end of the current list. If this is the case, ensure that there's another mechanism in place to handle this functionality.
```diff
- onMomentumScrollEnd={(event) => onWillSendViewPostTime(event, feeds)}
+ onMomentumScrollEnd={(event) => {
+   onWillSendViewPostTime(event, feeds);
+   fetchNextFeeds(event);
+ }}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->